### ISSUE TITLE
Backward support?

### ISSUE THREAD

**Issue by QiuJueqin** [cheind/pytorch-debayer#2]
Hi, have you ever tried to integrate this debayer module with other trainable module (nn.Module)? Will the back propagation work well for the debayer module?

**Comment by cheind** [comment]
Hey! At least for the 2x2 and 3x3 I don't see why gradient flow should be blocked. Note however, the parameters of debayer layer are not learnable per se.

**Comment by QiuJueqin** [comment]
> per se

Yes, making kernel in demosaic unaltered is reasonable. Thanks!
